Makefile

```diff
@@ -48,3 +48,6 @@ $(TARGETS) : % : $(addsuffix %,$(SUBDIRS))
 
 $(SUBDIRS_TARGETS) :
 	$(MAKE) -C $(@D) $(@F:.%=%)
+
+test:
+	go test -v ./...
```

Comment:
@michelleN this is going to cause an issue due to the fact that it's already defined above.